### ISSUE TITLE
Mention Support?

### ISSUE THREAD

**Issue by homerlex** [nozer/quill-delta-to-html#11]
I see that functionality was added to this project to support mentions.  Is there a quill mention module that is compatible with the mentions code in here?

**Comment by nozer** [comment]
@buildAI made a pull request and I merged without questioning partly because he included test and everything and I did not want to disrespect his effort. I asked him the same question now; I will let you know once he answers.

**Comment by buildAI** [comment]
@nozer @homerlex I did a fork of this project and tried to  make a custom mentions blot but wasn't really successful just was digging through emails and noticed my PR was merged into this. I would say there wasn't any module. I just added an ops support which I thought was generic enough when I was working on custom module. Let me know if I need to add on to this but I'm glad this was merged @nozer this is my first contribution to public repo. Let me know what I can do make things better.
